contracts/.solhint.json

```diff
@@ -3,11 +3,10 @@
   "plugins": ["prettier"],
   "rules": {
     "code-complexity": ["error", 7],
-    "compiler-version": ["error", "^0.6.12"],
+    "compiler-version": ["warn", "^0.6.12"],
     "const-name-snakecase": "off",
     "constructor-syntax": "error",
```

Comment:
(arbitrary location)

There seems to be an edge case when you have both a long network name (Arbibtrum One) and a long ENS name, the wrapping gets weird and thrown off for both of them. Not sure how hard of a fix this one is but if it's possible to fix easily we should— otherwise issue is fine.

<img width="399" alt="Screen Shot 2022-03-25 at 4 51 28 PM" src="https://user-images.githubusercontent.com/1479802/160201541-06f4aadd-d504-47fe-b4f5-2c9b28330fb0.png">

Reply:
Just pushed a81667f which makes the header columns floating width instead of fixed fractions of browser width. Seems to fix the issue for me